<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SaverGui</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>394</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>FO login</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="le_login" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QPushButton" name="pbsave" >
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Dw size</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2" >
     <widget class="QSpinBox" name="sb_dwsize" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="suffix" >
       <string> kB</string>
      </property>
      <property name="minimum" >
       <number>1000</number>
      </property>
      <property name="maximum" >
       <number>10000</number>
      </property>
      <property name="singleStep" >
       <number>500</number>
      </property>
      <property name="value" >
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="6" >
     <widget class="QPushButton" name="pbstart" >
      <property name="text" >
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>FO password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="le_password" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QCheckBox" name="cb_recheck" >
      <property name="text" >
       <string>Recheck</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Pagenum</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" >
     <widget class="QLCDNumber" name="pagenum" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>1</number>
      </property>
      <property name="midLineWidth" >
       <number>0</number>
      </property>
      <property name="smallDecimalPoint" >
       <bool>false</bool>
      </property>
      <property name="mode" >
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle" >
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="6" >
     <widget class="QPushButton" name="pbstop" >
      <property name="text" >
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="7" >
     <widget class="QListWidget" name="log" >
      <property name="lineWidth" >
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
